src/cpp/session/modules/SessionFiles.R

```diff
@@ -174,3 +174,19 @@ for (binding in bindings)
 {
    .Call("rs_readLines", path.expand(filePath))
 })
+
+.rs.addJsonRpcHandler("get_issue_url", function(id)
```

Comment:
Maybe this could be done in the client. Do we have the content from the DESRIPTION handy somewhere ?

Reply:
I don't think we cache this information on the client -- part of the motivation there being that the DESCRIPTION file might change while the project is open, and so we'd need to keep the client-side copy of that up-to-date?

We could do that if necessary but (at least thus far) it's been easier to just request the information as we need it.